### ISSUE TITLE
Needs updated versions of CDNs

### ISSUE THREAD

**Issue by siakaramalegos** [starterleague/starter_generators#31]
I don't mind doing a pull request but didn't want to start working on it if you aren't really active anymore.  Let me know!

**Comment by JeffCohen** [comment]
I don't think anyone's maintaining this repo any longer.  Feel free to fork and maintain there instead.

**Comment by siakaramalegos** [comment]
Yeah, I was thinking of doing that but thought I would check in first.  :)

**Comment by siakaramalegos** [comment]
@derencius if you want to transfer the gem over officially on rubygems, I don't mind taking over.
